calculator.js

```diff
@@ -0,0 +1,18 @@
+//Calculator!
+var calc = {
+    add: function(num1, num2) {
+        return num1 + num2},
+    subtract: function(num1, num2) {
+        return num1 - num2},
+    multiply: function(num1, num2) {
+        return num1 * num2},
```

Comment:
Make sure you test your code, Sanjay! This would all error because you didn't include closing `}` for each method.